RoosterPlanner.Api/AutoMapperProfiles/AutoMapperProfile.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using AutoMapper;
 using RoosterPlanner.Api.Models;
+using RoosterPlanner.Models.Models;
```

Comment:
Namespace van de models in project RoosterPlanner.Models moet RoosterPlanner.Models zijn. De extra 'Models' deel van de folder waarin deze staan mag weggehaald worden.

Reply:
@DRTomberge Momenteel staan ze echter wel in de map: RoosterPlanner.Models.Models. (er is ook een folder RoosterPlanner.Models.FilterModels). Wil je dat ik de inhoud van beide mappen verplaats naar Roosterplanner.Models?

Reply:
Nee, alleen de namespace van de classes van de models aanpassen. Die van RoosterPlanner.Models.FilterModels kan zo blijven.